modules/plot/main/coffee/Series.coffee

```diff
@@ -1,7 +1,7 @@
 
 class Series extends hx.EventEmitter
 
-  defaultLabelValuesExtractor = (series, dataPoint, xAccessor, yAccessor, xProperty, yProperty) ->
+  defaultLabelValuesExtractor = (series, dataPoint, xAccessor, yAccessor, xProperty='x', yProperty='y') ->
```

Comment:
After chasing this down, I could only see [one place](https://github.com/ocadotechnology/hexagonjs/blob/1-17-0/modules/plot/main/coffee/Utils.coffee#L272) where these values are actually passed through by the code. This meant that all series using the default were doing `series.labelFormatter(undefined)` instead of `series.labelFormatter('x')` and essentially ignoring the `labelFormatters` option altogether